docs/dev/policy_documents/merging-and-promoting-conventions.md

```diff
@@ -0,0 +1,40 @@
+# Merging and Promoting Conventions
+
+## Summary
+[summary]: #summary
+
+This policy defines a 'fast-track' merge and promote policy for a subset of core-plans.  
+
+## Motivation
+[motivation]: #motivation
+
+If someone who contributes heavily to core plans, has a pull request for a minor change open for 7days +, with all tests passing, it would be good if the same person can also merge and promote if there is a lack of peer reviews. One contributor (Graham Wilson, predominant) has hundreds of updates in his backlog, but has been holding back because of delays in the review process.
+
+## Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+What plan is eligable for the fast-track criteria:
+
+* Only identified 'veteran' core-plan contributor's have permission to use this fast-track policy.  This list will be agreed and specified in the MAINTAINERS.md
+* This fasttrack will only apply to non-baseplan PRs and PRs that do not have a label 'DO-NOT-MERGE' or are otherwise marked for caution.
+* If no feedback has been added to the PR after 7+ days, then the veteran contributor can merge and promote the PR.
+
+## Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Delete this section?
+
+## Drawbacks
+[drawbacks]: #drawbacks
+
+What are the drawbacks if any?
```

Comment:
Drawback include (1) less awareness amongst maintainers about what changes have occurred recently, (2) opportunity for bugs to enter that would have been caught in code review, (3) decreases pressure to improve the code-review process that has led to long delays.

Reply:
Updating according to these comments